scripts/ci/run-jenkins.sh

```diff
@@ -0,0 +1,99 @@
+if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\Win32\\\bin\\\Release_SGen\\\mono-sgen.exe`";fi
+if [[ ${label} == 'w64' ]]; then PLATFORM=x64; export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\x64\\\bin\\\Release_SGen\\\mono-sgen.exe`"; fi
+
+${TESTCMD} --label=configure --timeout=60m --fatal ./autogen.sh --with-monodroid --with-monotouch --with-monotouch_watch --with-monotouch_tv --with-xammac --with-mobile_static $EXTRA_CONF_FLAGS
+if [ $? -ne 0 ]; then exit 1; fi
+if [[ ${label} == w* ]];
+    then
+    ${TESTCMD} --label=make-msvc --timeout=60m --fatal /cygdrive/c/Program\ Files\ \(x86\)/MSBuild/12.0/Bin/MSBuild.exe /p:Platform=${PLATFORM} /p:Configuration=Release msvc/mono.sln
+    if [ $? -ne 0 ]; then exit 1; fi
+    ${TESTCMD} --label=make-msvc-sgen --timeout=60m --fatal /cygdrive/c/Program\ Files\ \(x86\)/MSBuild/12.0/Bin/MSBuild.exe /p:Platform=${PLATFORM} /p:Configuration=Release_SGen msvc/mono.sln
+    if [ $? -ne 0 ]; then exit 1; fi
+fi
+${TESTCMD} --label=make --timeout=300m --fatal make -w V=1
+if [ $? -ne 0 ]; then exit 1; fi
+${TESTCMD} --label=mini --timeout=5m make -w -C mono/mini -k check
+${TESTCMD} --label=runtime --timeout=120m make -w -C mono/tests -k test-wrench V=1 CI=1
+${TESTCMD} --label=corlib-4.5 --timeout=30m make -w -C mcs/class/corlib run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=verify --timeout=15m make -w -C runtime mcs-compileall
+${TESTCMD} --label=profiler --timeout=30m make -w -C mono/profiler -k check
+${TESTCMD} --label=compiler-4.5 --timeout=30m make -w -C mcs/tests run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=compiler-errors-4.5 --timeout=30m make -w -C mcs/errors run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=System-4.5 --timeout=10m make -w -C mcs/class/System run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=System.XML-4.5 --timeout=5m make -w -C mcs/class/System.XML run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=Mono.Security-4.5 --timeout=5m make -w -C mcs/class/Mono.Security run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=System.Security-4.5 --timeout=5m make -w -C mcs/class/System.Security run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+${TESTCMD} --label=System.Drawing-4.5 --timeout=5m make -w -C mcs/class/System.Drawing run-test TEST_HARNESS_FLAGS=-labels PROFILE=net_4_x
+if [[ ${label} == osx-* ]]
+then echo -e "*** start: Windows.Forms-4.5\n*** end(0): Windows.Forms-4.5: \e[45mSkipped\e[0m"
```

Comment:
Let's make this a separate parameter (`--skip`?) of run-test.sh so we have this completely decoupled.